clustering/server/src/main/java/org/wildfly/clustering/server/singleton/DistributedSingletonService.java

```diff
@@ -83,7 +83,7 @@ public DistributedSingletonService(DistributedSingletonServiceContext<T> context
     @Override
     public void start(StartContext context) throws StartException {
         ServiceTarget target = context.getChildTarget();
-        this.primaryController = target.addService(this.serviceName.append("primary"), this.primaryService).setInitialMode(ServiceController.Mode.ON_DEMAND).install();
+        this.primaryController = target.addService(this.serviceName.append("primary"), this.primaryService).setInitialMode(ServiceController.Mode.NEVER).install();
         this.backupController = target.addService(this.serviceName.append("backup"), this.backupService.orElse(new PrimaryProxyService<>(this))).setInitialMode(ServiceController.Mode.PASSIVE).install();
```

Comment:
Tangentially, we should likewise install the backup service as ACTIVE, so that it toggles to NEVER instead of ON_DEMAND as well.